### PR TITLE
Add the newsletter optin to edit customer page

### PR DESCRIPTION
**Pull request by fatmaBouchekoua**
Add the newsletter option to edit customer page.

Fix ticket: http://forge.prestashop.com/browse/BOOM-1317

### CODE REVIEW

**Comment by vincentbz** [comment]
Tested and ok, thanks !

File: ps_emailsubscription.php [hunk 1]
```diff
@@ -97,6 +97,7 @@ public function install()
                     'displayFooterBefore',
                     'actionCustomerAccountAdd',
                     'additionalCustomerFormFields',
+                    'displayAdminCustomersForm',
```
Comment (eternoendless) [inline ps_emailsubscription.php]: The permissions for this file have been modified
screen shot 2017-08-17 at 11 31 06

File: ps_emailsubscription.php [hunk 2]
```diff
@@ -1214,4 +1215,41 @@ private function getConditionFixtures($lang)
             $this->trans('You may unsubscribe at any moment. For that purpose, please find our contact info in the legal notice.', array(), 'Modules.Emailsubscription.Shop', $locale)
         ;
     }
+
+    /**
+     * hook display AdminCustomers form
```
Comment (eternoendless) [inline ps_emailsubscription.php]: This doesn't really add much information, please write a more precise description.

File: ps_emailsubscription.php [hunk 3]
```diff
@@ -1214,4 +1215,41 @@ private function getConditionFixtures($lang)
...
+    {
+        $newsletter = Db::getInstance(_PS_USE_SQL_SLAVE_)->getValue('SELECT `newsletter`
+            FROM ' . _DB_PREFIX_ . 'customer
+            WHERE `id_customer` = ' . (int)Tools::getValue('id_customer', 0));
+
+        $input = array(
+            'type' => 'switch',
+            'label' => $this->trans('Newsletter', array(), 'Admin.Orderscustomers.Feature'),
+            'name' => 'newsletter',
+            'required' => FALSE,
```
Comment (eternoendless) [inline ps_emailsubscription.php]: `FALSE` and `TRUE` should always be written in minuscule as per PSR-2

File: views/templates/admin/newsletter_subscribe.tpl [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+*}
+{block name="input_row"}
+  <div class="form-group">
+    {block name="label"}
+      <label class="control-label col-lg-3{if isset($input.required) && $input.required} required{/if}">
+          <span class="label-tooltip" data-toggle="tooltip" data-html="true" title="{if isset($input.hint)}{$input.hint|escape:'html':'UTF-8'}{/if}">
+            {if isset($input.label)}
+              {$input.label}
+            {/if}
+  	      </span>
```
Comment (eternoendless) [inline views/templates/admin/newsletter_subscribe.tpl]: This span is wrongly indented

File: views/templates/admin/newsletter_subscribe.tpl [hunk 5]
```diff
@@ -0,0 +1,60 @@
+{*
+* 2007-2016 PrestaShop
```
Comment (maximebiloe) [inline views/templates/admin/newsletter_subscribe.tpl]: Wrong licence header

File: views/templates/admin/newsletter_subscribe.tpl (hunk 4 above)
Comment (fatmaBouchekoua) [inline views/templates/admin/newsletter_subscribe.tpl]: It's well indented on my local file. I don't know why it's not the case here !

Comment (eternoendless) [inline views/templates/admin/newsletter_subscribe.tpl]: There are mixed tabs and spaces on line 33, that's why 😉

Comment (fatmaBouchekoua) [inline views/templates/admin/newsletter_subscribe.tpl]: Right ! Done

Thanks!

**Comment by mickaelandrieu** [comment]
Note: I can't dismiss current reviews or add ``Waiting for QA feedback``, I don't have the rights on this repository.

**Pull request merged**